src/explore-education-statistics-common/src/modules/table-tool/components/WizardStepFormActions.tsx

```diff
@@ -55,7 +55,9 @@ const WizardStepFormActions = ({
         </Button>
       )}
 
-      {form.isSubmitting && <LoadingSpinner inline size={39} />}
+      {form.isSubmitting && (
+        <LoadingSpinner text="Loading..." inline size={39} />
```

Comment:
I think this should use the screenReaderMessage attribute instead so it hides the text visually